.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test and update
+
+on:
+  schedule:
+    - cron: "37 6 * * *"
```

Comment:
This is just some random time, 6:37AM UTC.

Reply:
Yup. Really- anytime is fine. On Heroku- it def ran for quite a while.